### PR TITLE
Fix: typos

### PR DESCRIPTION
**Pull request by omahs**
Fix: typos

### CODE REVIEW

File: docs/the_nimbus_book/src/metrics-pretty-pictures.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 In this page we'll cover how to use  Grafana and Prometheus to help you visualise important real-time metrics concerning your validator and/or beacon node.
 
-Prometheus is an open-source systems monitoring and alerting toolkit. It runs as a service on your computer and its job is to capture metrics. You can find more information about Prometheus [here](https://prometheus.io/docs/introduction/overview/).
+Prometheus is an open-source system monitoring and alerting toolkit. It runs as a service on your computer and its job is to capture metrics. You can find more information about Prometheus [here](https://prometheus.io/docs/introduction/overview/).
```
Comment (tersec) [inline docs/the_nimbus_book/src/metrics-pretty-pictures.md]: Both of these are defensible, and arguably "systems monitoring" is more applicable if someone's using multiple machines to host their node infrastructure.

**Comment by tersec** [comment]
Seems mostly fine, but should target the `unstable` branch, which is where ongoing development occurs.

**Comment by omahs** [comment]
@tersec ok! I just did the change

File: docs/the_nimbus_book/src/metrics-pretty-pictures.md (hunk 1 above)
Comment (omahs) [reply docs/the_nimbus_book/src/metrics-pretty-pictures.md]: Ok, I cancelled this change 👍

**Review by tersec — APPROVED** [review]

**Pull request merged**